sdk/communication/communication-verified-recipients/samples-dev/deleteVerification.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Delete a verification for a resource
+ */
+
+import { VerifiedRecipientsClient } from "@azure-tools/communication-verified-recipients";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export async function main() {
+  console.log("\n== Delete Verification Sample ==\n");
+
+  // You will need to set this environment variable or edit the following values
+  const connectionString =
+    process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING ||
+    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
+
+  // create new client
+  const client = new VerifiedRecipientsClient(connectionString);
+
+  // id that is used to reference users phone number
+  const verificationId = "4d313ff0-3aeb-477e-8c15-7c9a893e8999";
```

Comment:
is this a random id or the one users have to use?
If it is something the users should get from the portal, you could make it an env variable too.

Reply:
oh thats a great idea!